model/modeldecoder/v2/decoder.go

```diff
@@ -41,6 +42,15 @@ type DecodeError struct {
 	err error
 }
 
+var jsoniterErrRegexp = regexp.MustCompile(`expect.* but found .*error found in .* bigger context.*`)
```

Comment:
What do you think about keeping the "expect <...>" bit intact? That could be helpful to understand why the field is invalid, and is based on the target field type rather than the input.

Reply:
yes, sure we can keep that